client/components/Settings.js

```diff
@@ -82,29 +90,38 @@ class ChangePasswordModal extends React.Component {
     }
 
     render() {
-        let modalTitle = <T k="changepasswordmodal.title">Change bank password</T>;
+        let customFields;
+
+        if (this.props.customFields) {
+            customFields = this.props.customFields.map((field, index) =>
+                <CustomBankField ref={"customField" + index} params={field} />
+            );
+        }
+
+        let modalTitle = <T k="editaccessmodal.title">Change bank password</T>;
```

Comment:
Oops, this shouldn't be "Change bank password" but "Edit bank access" or something else. (won't prevent merging)